beautifultable/beautifultable.py

```diff
@@ -1213,3 +1216,47 @@ def from_csv(self, file_name, header=True, **kwargs):
             for row in csv_reader:
                 self.rows.append(row)
             return self
+
+    def to_df(self):
+        """Export table to dataframe.
+
+        Parameters
+        ----------
+        table : Beautifuletable
```

Comment:
typo

Reply:
Resolved

Reply:
It seems you changed it to **BeautifullTable**. it should be **BeautifulTable**